_pages/meetings/spring2021.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: page
+title: 2021 Spring Python in Heliophysics Community Meeting
+permalink: /meetings/spring2021/
+exclude: true
+---
+
+The spring 2021 meeting of the [Python in Heliophysics Community](http://heliopython.org) will be held remotely [**at this Zoom link**](https://cuboulder.zoom.us/j/395871054) (Meeting ID: **395 871 054**). It will be spread across four sessions, starting Monday, May 10th, 2021 and ending Thursday, May 13th, 2021:
+
+ - **Presentations:** May 10th 9:00 AM–11:00 AM (MT)
+ - **Tutorials/Presentations:** May 11th 9:00 AM–11:00 AM (MT)
+ - **Hackathons:** May 12th 9:00 AM–11:00 AM (MT)
+ - **Discussions:** May 13th 9:00 AM–11:00 AM (MT)
+
+See [the full agenda](https://docs.google.com/spreadsheets/d/1zyakyCm6tnahqjHZHZchYxPrq8mf5FYIY_U3bvIpHNg/edit#gid=0) for exact dates/times of the various sessions. 
+
+### Session Documents and Presentations
+
+Materials associated with the meeting sessions will be kept in the [**meeting folder**](https://drive.google.com/drive/u/0/folders/1HcIQRnVmEXiTgNVx7cVL5mMySxVbUFYc) on Google Drive.
+
+ - [Meeting schedule](https://docs.google.com/spreadsheets/d/1zyakyCm6tnahqjHZHZchYxPrq8mf5FYIY_U3bvIpHNg/edit#gid=0) (subject to change)
+ - [Hackathon projects](https://docs.google.com/spreadsheets/d/1EUqKfj6iAEPj2AQsyeNyo03qlbCFL0jNoygja4N6KtQ/edit#gid=0)
+ - [Presentation and Tutorials Organizing](https://docs.google.com/spreadsheets/d/1JKoWtBTkwsackg_DbFFb3EXHXNDko0mmBBTu9cc2A2g/edit#gid=0)
+
+### Zoom Meeting Information
+
+Join Zoom Meeting
+[**https://cuboulder.zoom.us/j/395871054**](https://cuboulder.zoom.us/j/395871054)
+
+Meeting ID: **395 871 054**
+
+One tap mobile
+ - +16699006833,,395871054# US (San Jose)
+ - +16465588656,,395871054# US (New York)
+
+Dial by your location
+ - +1 669 900 6833 US (San Jose)
+ - +1 646 558 8656 US (New York)
+
+Meeting ID: **395 871 054**
+
+Find your local number: https://cuboulder.zoom.us/u/ab3XRN96S2
+
+Join by SIP
+ - 395871054@zoomcrc.com
+
+Join by H.323
+ - 162.255.37.11 (US West)
+ - 162.255.36.11 (US East)
+ - 221.122.88.195 (China)
+ - 115.114.131.7 (India)
+ - 213.19.144.110 (EMEA)
+ - 103.122.166.55 (Australia)
+ - 209.9.211.110 (Hong Kong)
+ - 64.211.144.160 (Brazil)
+ - 69.174.57.160 (Canada)
+ - 207.226.132.110 (Japan)
+ 
+Meeting ID: **395 871 054**  
+
+### Registration
+
+Please [**register**](https://docs.google.com/forms/d/1tk9uQTm9TzwNV8jle3QCg8IZkhZsQjLKMQDN-02a4IY/edit?usp=sharing) by close of business on October 23, 2020 if you plan to participate remotely so that you may receive email announcements and updates.  No registration fee is required.  You may still participate without registering.
```

Comment:
This should say COB, Friday May 7th, 2021 as the registration due date (instead of Oct 23 2020).